app/tests/api/db/models/factories.py

```diff
@@ -58,25 +58,25 @@ class Meta:
         sqlalchemy_session_persistence = "commit"
 
 
-class UserRoleFactory(BaseFactory):
+class RoleAssignmentFactory(BaseFactory):
```

Comment:
Should this just be `RoleFactory` so it's just `{OriginalClass}Factory`?

Reply:
Yeah good catch, I missed it during my renames